Tests/BuildTests/PluginInvocationTests.swift

```diff
@@ -810,6 +810,177 @@ final class PluginInvocationTests: XCTestCase {
         }
     }
 
+    func testPrebuildPluginShouldUseBinaryTarget() async throws {
+        // Only run the test if the environment in which we're running actually supports Swift concurrency (which the plugin APIs require).
+        try XCTSkipIf(!UserToolchain.default.supportsSwiftConcurrency(), "skipping because test environment doesn't support concurrency")
```

Comment:
Note: this is unnecessary / dead code, SwiftPM requires macOS 13 now, which always supports concurrency.